test/unit/services/test_catalogue_item.py

```diff
@@ -604,15 +635,23 @@ def test_update_when_no_child_elements(test_helpers, catalogue_item_repository_m
         id=str(ObjectId()),
         catalogue_category_id=str(ObjectId()),
         manufacturer_id=str(ObjectId()),
-        **FULL_CATALOGUE_ITEM_A_INFO,
+        **{
+            **FULL_CATALOGUE_ITEM_A_INFO,
+            "created_time": FULL_CATALOGUE_ITEM_A_INFO["created_time"] - timedelta(days=5),
```

Comment:
why is there this `- timedelta(days=5)` in a lot of the tests? I don't understand what its purpose is

Reply:
Both create and update times are generated when something is created. In the update tests, we return what the entity looks like before it is updated so we set the create and modified time 5 days earlier so that we can then ensure that the modified time gets updated to the mocked datetime. Let me know if this does not make sense and I will talk to you.